pkgs/applications/graphics/weylus/default.nix

```diff
@@ -9,6 +9,9 @@
 , copyDesktopItems
 , fontconfig
 , libpng
+, pipewireSupport ? true
```

Comment:
Why would you want to disable this?

Does this package build on darwin?

Also why is there no meta.platforms set.

Reply:
> Why would you want to disable this?

I suppose you wouldn't. Removed arg.

> Does this package build on darwin?
>
> Also why is there no meta.platforms set.

It *should*, however I don't have a machine to test on and the current derivation grabs the Linux binary in place of building it from source; it can be a future improvement. Added meta.platforms for x86_64 linux.